sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample7_AnalyzeHealthcareEntitiesBatch.cs

```diff
@@ -17,12 +17,13 @@ public partial class TextAnalyticsSamples: SamplesBase<TextAnalyticsTestEnvironm
         [Ignore("Healthcare not enabled yet")]
```

Comment:
wow! not related to your PR but I just noticed that we are not running these samples!!!! :|
I created issue https://github.com/Azure/azure-sdk-for-net/issues/21471 so we can enable them or at least understand why they are disabled

Reply:
@maririos 
yeah, i was gonna ask you about this "[Ignore("Healthcare not enabled yet")]"